install/kubernetes/helm/istio/charts/tracing/templates/deployment-jaeger.yaml

```diff
@@ -83,11 +73,9 @@ spec:
             httpGet:
               path: /
               port: 16686
-{{- if eq .Values.jaeger.spanStorageType "badger" }}
```

Comment:
Isn't ideal for the volume mounts and volumes - is there a way to check the value of a map entry, i.e. `eq .Values.jaeger.env['SPAN_STORAGE_TYPE'] "badger"`?

Reply:
I am not sure. But also, keep in mind that this is essentially mounting an empty directory at the cost of a big simplification in the UX and flexibility. No persistent volumes,etc.

Reply:
Fair point.